modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts

```diff
@@ -534,10 +536,12 @@ export class EddsaUtils extends baseTSSUtils<KeyShare> {
       publicShare
     );
 
+    const bitgoToUserCommitment = {} as Commitment; // TODO: Not sure where to source this from -JD
```

Comment:
Probably a txrequest endpoint to retrieve it from BitGo or as a response to the last step?, Need to finalise the API request flow for this if not discussed previously